### ISSUE TITLE
tkinter.filedialog.askopenfilename doesn't exist?

### ISSUE THREAD

**Issue by rbalik** [PythonCharmers/python-future#212]
I'm not sure if this is an issue in python-future or what, but the tkinter that comes with python-future is missing askopenfilename.

This is on Windows:

```
$ python
Python 2.7.11 (v2.7.11:6d1b6a68f775, Dec  5 2015, 20:40:30) [MSC v.1500 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import tkinter.filedialog
>>> tkinter.filedialog.askopenfilename
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'askopenfilename'
>>>
```

On an actual python3 install, that second command doesn't fail.

**Comment by gpip** [comment]
It's a python-future issue, `tkinter.filedialog` is missing a `from tkFileDialog import *` right after `from FileDialog import *` for Python 2.

**Comment by edschofield** [comment]
Thanks for your feedback, Reuben and Guilherme!

This should be fixed in v0.16.0 now with commit 5cd9587d21d04443ded94244caf0474cef0c6a21. Please re-open this issue if you notice any related problems.
